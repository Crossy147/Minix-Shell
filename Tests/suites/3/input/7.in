# multiple pipelines per line


bin/decho 2 a b c | cat  > f1 ; cat f1 | sort 

ls servers > f1 ; cat f1 > f2 ; cat f2 > f3 ; cat f3 > f4 ; cat f4

rm f1 f2 f3 f4

