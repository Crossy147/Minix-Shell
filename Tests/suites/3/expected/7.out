 a b c
Makefile
fs
init
pm
rs
